spec/client_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'spec_helper'
+
+describe 'rdiff-backup::client' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
```

Comment:
Needs to test line 45
`node.override['ssh_keys'][client_user] = ['rdiff-backup-client']`
example: https://github.com/sethvargo/chefspec/blob/master/examples/attributes/spec/default_spec.rb